apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/familieforhold/omstillingsstoenad/BarneListe.tsx

```diff
@@ -69,44 +60,34 @@ const BarnRow = ({ barn, familieforhold }: { barn: IPdlPerson; familieforhold: F
           {barn.fornavn} {barn.etternavn} <DoedsdatoTag doedsdato={barn.doedsdato} />
         </Table.DataCell>
         <Table.DataCell>
-          <FnrWrapper>
-            <Link href={`/person/${barn.foedselsnummer}`} target="_blank" rel="noreferrer noopener">
-              {formaterFnr(barn.foedselsnummer)}
-            </Link>
-            <CopyButton copyText={barn.foedselsnummer} size="small" />
-          </FnrWrapper>
+          <KopierbarVerdi value={barn.foedselsnummer} iconPosition="right" />
         </Table.DataCell>
         <Table.DataCell>-</Table.DataCell>
         <Table.DataCell>-</Table.DataCell>
+        <Table.DataCell>-</Table.DataCell>
         <Table.DataCell>{erGjenlevendesBarn ? 'Gjenlevende og avdød' : 'Kun avdød'}</Table.DataCell>
       </Table.Row>
     )
   }
 
-  const alder = hentAlderForDato(barn.foedselsdato)
   const aktivAdresse: IAdresse | undefined = barn.bostedsadresse?.find((adresse: IAdresse) => adresse.aktiv)
   const adresse = `${aktivAdresse?.adresseLinje1}, ${aktivAdresse?.postnr ?? ''} ${aktivAdresse?.poststed ?? ''}`
-  const periode = aktivAdresse
-    ? `${formaterKanskjeStringDato(aktivAdresse.gyldigFraOgMed)} - ${
-        aktivAdresse?.gyldigTilOgMed ? formaterDato(aktivAdresse!!.gyldigTilOgMed!!) : 'nå'
-      }`
-    : 'Mangler adresse'
 
   return (
     <Table.Row>
       <Table.DataCell>
-        {barn.fornavn} {barn.etternavn} ({alder} år)
+        {barn.fornavn} {barn.etternavn} ({hentAlderForDato(barn.foedselsdato)} år)
       </Table.DataCell>
       <Table.DataCell>
-        <FnrWrapper>
-          <Link href={`/person/${barn.foedselsnummer}`} target="_blank" rel="noreferrer noopener">
-            {formaterFnr(barn.foedselsnummer)}
-          </Link>
-          <CopyButton copyText={barn.foedselsnummer} size="small" />
-        </FnrWrapper>
+        <KopierbarVerdi value={barn.foedselsnummer} iconPosition="right" />
       </Table.DataCell>
       <Table.DataCell>{adresse}</Table.DataCell>
-      <Table.DataCell>{periode}</Table.DataCell>
+      <Table.DataCell>
+        {!!aktivAdresse ? formaterDatoMedFallback(aktivAdresse.gyldigFraOgMed, '-') : 'Mangler adresse'}
+      </Table.DataCell>
+      <Table.DataCell>
+        {!!aktivAdresse ? formaterDatoMedFallback(aktivAdresse.gyldigTilOgMed, '-') : 'Mangler adresse'}
```

Comment:
Mangler vel også en fallback for "nå" hvis tom mangler? Nå vil det vel så "fom" - tom streng?